src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -94,6 +94,7 @@ class ScalaSettings extends Settings.SettingGroup {
   val sourceReader = StringSetting("-Xsource-reader", "classname", "Specify a custom method for reading source files.", "")
   val XnoValueClasses = BooleanSetting("-Xno-value-classes", "Do not use value classes. Helps debugging.")
   val XreplLineWidth = IntSetting("-Xrepl-line-width", "Maximial number of columns per line for REPL output", 390)
+  val XreplNoColor = BooleanSetting("-Xrepl-no-color", "Removes syntax highlighting in REPL")
```

Comment:
@smarter: It felt like the precedence was to name the option `-Xrepl-no-color` as opposed to the `-Dscala.color` in scalac

(Other suggested changes in place)